src/Dotnet.Script.Core/Scaffolder.cs

```diff
@@ -141,7 +141,7 @@ private void CreateLaunchConfiguration(string currentWorkingDirectory)
             _scriptConsole.WriteNormal("Creating VS Code launch configuration file");
             string pathToLaunchFile = Path.Combine(vsCodeDirectory, "launch.json");
             string installLocation = _scriptEnvironment.InstallLocation;
-            bool isInstalledAsGlobalTool = installLocation.Contains(".dotnet/tools", StringComparison.OrdinalIgnoreCase);
+            bool isInstalledAsGlobalTool = installLocation.Contains($".dotnet{Path.DirectorySeparatorChar}tools", StringComparison.OrdinalIgnoreCase);
             string dotnetScriptPath = Path.Combine(installLocation, "dotnet-script.dll").Replace(@"\", "/");
```

Comment:
@seesharper why does this normalise everything to forward-slash, instead of using the platform-specific separator?

Reply:
it doesn't - `DirectorySeparatorChar` resolves to the platform specific separator